README.md

```diff
@@ -2,3 +2,80 @@ mobile-cli-lib
 ==============
 
 Contains common infrastructure for CLIs - mainly AppBuilder and NativeScript.
+
+Public API
+==
+
+This section contains information about each public method. All methods return Promise.
+
+### How to make a method public
+In order to expose public API, we use TypeScript decorators and some "magic" in our bootstrapping. When you want to expose method `B` from class `A`, you have to do the following:
+* In `bootstrap.ts` make sure to use `requirePublic` method of the `$injector` when requiring the class:
+	```TypeScript
+	$injector.requirePublic("a", "./pathToFileWhereAClassIs")
```

Comment:
Consider reordering the help so that explanations about exporting and decorators come before the `$injector.requirePublic` part.